docs/tutorial/getting-started-with-kiali.md

```diff
@@ -0,0 +1,162 @@
+# Getting Started with Kiali
+
+This tutorial demonstrates how to deploy and configure [Kiali](https://kiali.io) to observe your Charmed Istio service mesh. You'll learn how to visualize your mesh topology, monitor traffic flow, and gain insights into your distributed applications.
+
+## Introduction
+
+Kiali is a management console for [Istio](https://istio.io) service mesh that provides visualization through a modern web interface. The `kiali-k8s` charm simplifies the deployment and integration of Kiali with [Charmed Istio](./getting-started-with-istio-mesh.md), offering:
+
+- Service mesh topology visualization
+- Traffic flow monitoring
+- Health status indicators
+- Configuration validation
+- Distributed tracing integration
+
+In this tutorial, you'll deploy Charmed Kiali to observe the Charmed Istio and the [Charmed Bookinfo](./getting-started-with-istio-mesh.md#deploy-charmed-bookinfo-application) application from the previous tutorials, enabling you to visualize and monitor your service mesh in real-time.
+
+## Prerequisites
+
+Before starting this tutorial, ensure you have:
+
+- **Completed Tutorials**:
+  - [Getting Started with Charmed Istio Service Mesh](./getting-started-with-istio-mesh.md)
+  - [Getting Started with Charmed Istio: Cross-Model Mesh](./getting-started-with-istio-cross-model-mesh.md)
+- **Working Deployment**: Charmed Istio deployed in the `istio-system` model
+- **Juju Knowledge**: Basic familiarity with [Juju cross-model relations](https://documentation.ubuntu.com/juju/stable/reference/cross-model-integration/)
+
+## Prepare the Istio System
+
+### Step 1: Deploy Prometheus
+
+Switch to the `istio-system` model and deploy Prometheus to collect metrics from Istio:
+
+```bash
+juju switch istio-system
+juju deploy prometheus-k8s --trust --channel=2/edge
```

Comment:
If we're pinning to a channel here, we should document why (and probably make an issue to remind us to fix it later)

Is this pin because track `1` doesn't have the prometheus-api relation?